apps/backend/example.development.env

```diff
@@ -50,6 +50,7 @@ JWT_SECRET=qMyLZALzs229ybdQXNyzYRdju7X784TH
 TZ=Europe/Stockholm
 DATE_FORMAT=dd-MM-yyyy
 DATE_TIME_FORMAT=dd-MM-yyyy HH:mm
+USER_OFFICE_EMAIL=useroffice1@email.com; useroffice2@email.com
```

Comment:
Is it possible for us to make this an array.

Reply:
I don't think so, it seems docker-compose and kubernetes only allows string values. You can separate them with a comma or some other delimiter and then parse in the code, but that's not much different to this. The benefit of using `;` here is that you can directly use the entire string without parsing in the to address of an email and it will send to all the addresses listed.

Reply:
I just need better clarification if this is user office email or user officer email.

Reply:
This is the central user office email. They wouldn't have an account because it doesn't belong to an individual. Also in our case we would additionally send these emails to User Officer Emma as requested. But we wouldn't want all User Officers to automatically be included here. Just thought I'd write down what we just discussed in the meeting 🙂